tests/src/hipEnvVarDriver.cpp

```diff
@@ -32,43 +32,74 @@ OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWA
 #include <thread>
 #include "test_common.h"
 
+
 using namespace std;
 
 int getDeviceNumber() {
     FILE* in;
     char buff[512];
-    string str;
+    const char* directed_dir;
+    const char* dir;
+    
+    if (_WIN64 == 1){
+        directed_dir = "directed_tests\\hipEnvVar -c";
+        dir = "hipEnvVar -c";
+    } else {
+        directed_dir = "./directed_tests/hipEnvVar -c";
+        dir = "./hipEnvVar -c";
+    }
```

Comment:
Define slash and backslash in test_common.h(.cpp) and use it here unconditionally.